odoo/openupgrade/doc/source/intro.rst

```diff
@@ -54,4 +54,5 @@ Migrating your database
 
    migrate.py
    migration_details
+   migration_pycharm
```

Comment:
Do you need to add a migration_pycharm.rst?

Reply:
yep sorry, two files were missing. thanks for checking so fast.